trafilatura/core.py

```diff
@@ -487,7 +488,7 @@ def extract_with_metadata(
         include_images: Take images into account (experimental).
         include_formatting: Keep structural elements related to formatting
             (only valuable if output_format is set to XML).
-        include_links: Keep links along with their targets (experimental).
+=        include_links: Keep links along with their targets (experimental).
```

Comment:
This can be removed ,right?

Reply:
@unsleepy22 Could you please remove it if necessary?

Reply:
OK, maybe mis-typed this.